product_assortment/models/res_partner.py

```diff
@@ -1,11 +1,19 @@
 # Copyright 2021 Tecnativa - Carlos Roca
+# Copyright 2021 Tecnativa - Carlos Dauden
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
-from odoo import models
+from odoo import api, fields, models
 
 
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
+    applied_assortments_ids = fields.Many2many(
```

Comment:
Please no plurals here for avoiding constant typos:

```suggestion
    applied_assortment_ids = fields.Many2many(
```